msctl

```diff
@@ -993,6 +993,54 @@ getServerCommand() {
     printf "$SERVER_LOCATION\n"
     exit 1
   fi
+  # Get the location of the Java virtual machine, use the default value if not provided.
+  DEFAULT_JAVAVM=$(getDefaultsValue "mscs-default-java" "$DEFAULT_JAVA")
+  JAVA=$(getMSCSValue "$1" "mscs-java" "$DEFAULT_JAVAVM")
+  if [ ! -e "$JAVA" ]; then
+    echo "ERROR: java not found!"
+    echo "Try installing this with:"
+    echo "sudo apt-get install default-jre"
+    exit 1
+  fi
+  # Verify that the Java virtual machine supports the server version.
+  JAVA_VERSION=$($JAVA -version 2>&1 | $PERL -ne 'if ($_ =~ /version\s"([\d\.]+)\.\d+\_\d+"/) { print $1 }')
+  case $JAVA_VERSION in
+    1.9)
+      # Java 9 supports all server versions.
+    ;;
+    1.8)
+      # Java 8 supports all server versions.
+    ;;
+    1.7)
+      # Support for Java 7 ended with version 1.12.
+      VERSION_TEST=$(compareMinecraftVersions $SERVER_VERSION 1.12.1)
+      if [ $VERSION_TEST -ge 0 ]; then
+        echo "Error: Server version $SERVER_VERSION not supported by Java 7."
+        exit 1
+      fi
+    ;;
+    1.6)
+      # Support for Java 6 ended with version 1.12.
+      VERSION_TEST=$(compareMinecraftVersions $SERVER_VERSION 1.12.1)
+      if [ $VERSION_TEST -ge 0 ]; then
+        echo "Error: Server version $SERVER_VERSION not supported by Java 6."
+        exit 1
+      fi
+    ;;
+    1.5)
+      # Support for Java 5 ended with version 1.6.
+      VERSION_TEST=$(compareMinecraftVersions $SERVER_VERSION 1.6.1)
+      if [ $VERSION_TEST -ge 0 ]; then
```

Comment:
Same as above. Last 1.5 release was 1.5.2, but this is less of an issue because snapshots are only avail for the 1.12 series.